src/components/OptionsList/BaseOptionsList.js

```diff
@@ -97,9 +98,13 @@ class BaseOptionsList extends Component {
      * @return {Component}
      */
     renderItem({item, index, section}) {
+        const updatedItem = item;
+        if (item.isLastMessageAttachment) {
+            updatedItem.alternateText = `[${Localize.translateLocal('common.attachment')}]`;
+        }
```

Comment:
What is the use of this variable? I think this should be not be picked by linter.

```suggestion
        if (item.isLastMessageAttachment) {
            item.alternateText = `[${Localize.translateLocal('common.attachment')}]`;
        }
```

Reply:
There was issue in `npm run lint`.

Reply:
ok then update the item object immutably.

Reply:
I tried to update the item object immutably with:
```
        const updatedItem = {...item};
        if (updatedItem.isLastMessageAttachment) {
            updatedItem.alternateText = `[${Localize.translateLocal('common.attachment')}]`;
        }
```
But the component does not re-render. So, the effect of changing language is only seen when report in LHN is changed.

Reply:
This is a good sign we are not doing this at the correct place. Let's retry to rerender the list when the locale is changed so that we get the correct value. This will remove the need for `isLastMessageAttachment` property.

Reply:
> This is a good sign we are not doing this at the correct place. Let's retry to rerender the list when the locale is changed so that we get the correct value. This will remove the need for `isLastMessageAttachment` property.

OH!!! Yes.Instead of making any changes in libs/actions/Report.js. We can use '[Attahcment]' in lastMessageText    with ReportUtils.isReportMessageAttahcment() ,i.e. `ReportUtils.isReportMessageAttachment(report.lastMessageText)` in OptionsListUtils.js to know if it is attachment. And use it to set value of alternatetext in OptionsListUtils.js itself.